code/game/machinery/camera/presets.dm

```diff
@@ -25,7 +25,7 @@ var/global/list/station_networks = list(
 var/global/list/engineering_networks = list(
 										NETWORK_ENGINE,
 										NETWORK_ENGINEERING,
-										//NETWORK_ENGINEERING_OUTPOST,	//VOREStation Edit: Tether has no Engineering Outpost,
+										NETWORK_ENGINEERING_OUTPOST,	//VOREStation Edit: This network is for exterior power lines

```

Comment:
Should probably make different network with appropriate name for that, since Engineering Outpost would still be misleading name for the network. Or just put cameras on Engineering network, which would be way easier solution, since all substation do that already.